### ISSUE TITLE
Timeline Request

### ISSUE THREAD

**Issue by evronm** [completium/archetype-lang#335]
Hey Devs,

There are currently 2 showstoppers for me in Archetype, which have caused me to pause development on my project.  First, there's the name space issue.  Then there's the issue of not being able to reference asset partitions from within other asset partitions to create a many to many type relationship.  I can work around the latter by using IterableBigMaps, but it's kind of ugly.  But there's simply no workaround for the former that doesn't create an irretrievable mess.  

So my question is, is there some kind of timeline for when this will be addressed.  I was told last month that the namespace thing would be addressed in November, which is almost over, and there haven't been any commits in a couple of months.  

In short, any timeline guidance would be really appreciated as I really want to get back to coding this thing, and I am trying to find out if I should wait or move over to e.g. Ligo (which I'm reluctant to do because I really like Archetype).

Thanks in advance.

**Comment by rognierbenoit** [comment]
Hello @evronm,

thank you for your feedback.

Indeed we are late on the arl import. We should have it by January. 

Regarding "not being able to reference asset partitions from within other asset partitions" could you illustrate that with a basic example so that we have a precise idea of your need ?

We have been busy with support and documentation, and on the dev side, we have been working on the TS binding of archetype (and still are) which is a game changer for all integration subjects.

Kind regards.

**Comment by evronm** [comment]
Well, the classic example of what I'd want to do is something like tagging (not actually what I'm trying to do but a simple example):

`Asset tags {
  idt: nat;
  tag: string;
}
Asset posts {
  idp: nat;
  title: string;
  content: string;
  tags:  partition<post_tags>;
}
Asset post_tags {
  p_tags: partition<tags>;
}
`
I just typed that in, so I'm not sure it will give the correct error, but that's the gist of the kind of thing I'm trying to accomplish.

Thanks again.
